### DISCUSSION TITLE
Why do the videos when they are created not contain the season and episode in the file name?

### DISCUSSION THREAD

**Discussion by Gotas10** [crunchy-labs/crunchy-cli#406] [Q&A]
Why do the videos when they are created not contain the season and episode in the file name? For example, the file name contains S01E01
When I download the videos they only have the name of the episode.
I consider this to be important since if I only want the audios and/or subtitles of the videos, when I extract the tracks, they will only have the name of the episode but I will not know which episode or season it belongs to.

**Comment by bytedream** [comment]
What command did you use?

**Reply by Gotas10** [reply]
This: crunchy-cli archive -a ja-JP -s es-419 -r worst -t 10 https://www.crunchyroll.com/es/watch/G7PU4J1MJ/my-senpai-is-small-and-cute

**Comment by AdrianEdelen** [comment]
You can create custom output templates that make the filename whatever you want them to be, this is covered in the readme.
